app/controllers/field_slip_job_trackers_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class FieldSlipJobTrackersController < ApplicationController
+  before_action :login_required
+
+  # This is only a JSON endpoint describing the status of a particular tracker.
+  def show
+    # should we raise an error if the tracker is not found?
```

Comment:
Seems reasonable to raise an error, but where would it show up?

Reply:
See Slack for a possible new strategy using ActionCable broadcasts.